binary_gentoo/internal/cli/tree_diff.py

```diff
@@ -43,6 +79,12 @@ def parse_command_line(argv):
 
     add_version_argument_to(parser)
 
+    parser.add_argument('--keywords',
+                        dest='keywords',
+                        default='',
+                        help='filter the list on these keywords; '
+                             'in case of multiple keywords a space-separated list can be provided '
+                             '(default: all keywords)')
```

Comment:
With my picky hat on:
- verb "filter" has the problem that filters can be inclusive or exclusive. how about using "ignore"/"exclude" or "limit to" or so for a verb?
- we should probably mention that support `~arch` and `arch` syntax, but not `**`
- let's drop `default='',` and `dest='keywords',` boilerplate

Reply:
PS: Let's maybe use `portageq envvar ACCEPT_KEYWORDS` to auto-fill "enrich" the default on Gentoo systems maybe, as done for most other variables already. What do you think?

Reply:
Done the changes in your first comment.
For your second commend, that is indeed a good idea but it raises the following question: on Gentoo systems, the default is the value of the system's ACCEPT_KEYWORDS. But on non-Gentoo systems, do we make the default value empty (i.e. don't filter the list based on keywords) or does the `--keywords` argument become mandatory (as is now for e.g. the `--cflags` argument in `gentoo-build`, which simply fails if `portageq` cannot be executed)? For simplicity, I would say the second option (mandatory argument), because a user will want to build for a specific architecture anyway. Do you agree?

Reply:
I thought about using [distro](https://github.com/python-distro/distro) earlier to check for Gentoo and only call `portageq` on Gentoo, but I felt that consistent behavior and keeping full control with the user were more important. So yes, mandatory sounds great :+1:

Reply:
PS: I wonder if we should start supporting `**` for "any and all arches" for consistent behavior. It also seems more practical to passing a long hard-to-get-complete list of explicit keywords to get all of them.

Reply:
Wouldn't supporting `--keywords "**"` be functionally equivalent to omitting the `--keywords` argument? Just omitting it seems like an easy way to get "any and all arches". I would argue that it is less confusing for the user to either support all the special keywords or none of them, instead of only allowing `**`.

Reply:
Scratch that last remark, since we just discussed it would be mandatory ... Let me think a little bit about it.

Reply:
I've found an IMHO quite elegant solution to be able to support all special keywords `**`, `*` and `~*`. I'll let you have a look in the next commit which includes the `portageq envvar ACCEPT_KEYWORDS` too.